t/config.t

```diff
@@ -157,6 +162,28 @@ subtest 'Test configuration override from file' => sub {
         ],
         'referers parsed correctly'
     );
+    my $untracked_assets_patterns = $app->config->{'assets/storage_duration'};
+
+    ok(ref $untracked_assets_patterns eq ref {}, "failed to read config for assets/storage_duration");
+    ok(scalar %$untracked_assets_patterns, "config is empty for assets/storage_duration");
```

Comment:
`ok(scalar %hash` is IMHO the weirdest possible way to check for an empty hash :D

Reply:
Yea, just don't do that. Check the number of keys instead.